api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -409,7 +405,7 @@ RdrWorkspaceDemographic toRdrWorkspaceDemographics(
    * @param entity
    * @param idList
    */
-  private void updateDBRdrExport(RdrEntity entity, List<Long> idList) {
+  public void updateDBRdrExport(RdrEntity entity, List<Long> idList) {
```

Comment:
nit: while you're here, I'd fix the CamelCase in the method name: should be updateDbRdrExport (per Google's guidelines which I always link to, https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)

Reply:
👍